docs/how-to/major-upgrades.rst

```diff
@@ -0,0 +1,231 @@
+======================================
+Upgrade MicroOVN across major versions
+======================================
+
+MicroOVN is released in tracks that signify which version of ``OVN`` it
+bundles (e.g. ``22.03/stable`` track comes with ``OVN 22.03``). These tracks
+will keep tracking the same major version and wont upgrade to next major
+version on their own. To upgrade to next major version of MicroOVN, you have
+to change the track which your snap follows.
+
+In this how-to, we'll upgrade a cluster with four members, running
+``MicroOVN 22.03``, to ``MicroOVN 24.03``.
+
+Prepare cluster for upgrade
+---------------------------
+
+We start by ensuring that **each** of our cluster members runs MicroOVN from a
+track that precedes the track to which we are upgrading, and that it has latest
+upgrades from this track.
```

Comment:
I wanted to highlight that upgrades can be done only from one LTS, directly to the next. Right now we have only one stable branch, so a mistake is unlikely to happen, but when a `26.03` comes out, this may stop someone from attempting `22.03` -> `26.03` upgrade.
Though if you still think that mistake like this is unlikely, I'm happy to drop this step. I do have a feeling that this how-to is bit too long.

Reply:
Thanks for the review, I pushed an update that should address every one of your comments except this one. If you think that highlighting requirement to be on previous branch is too much here, I'll drop it. I can see how it perhaps tries to be too detailed.